termedia-neuropsychiatria-i-neuropsychologia-neuropsychiatry-and-neuropsychology.csl

```diff
@@ -94,12 +98,6 @@
       </if>
     </choose>
   </macro>
-  <locale>
-    <terms>
-      <term name="page-range-delimiter">-</term>
-      <term name="et-al"> i wsp.</term>
```

Comment:
If we remove language PL how this one line will work? This should apply to PL language only. I'm not super fluent in CSL for zotero, sorry if this one is stupid question :).
The other option to removing language PL is preparing two versions of styles one for PL articles and other one for ENG.

Reply:
@adam3smith